app.rb

```diff
@@ -1,9 +1,36 @@
 require 'sinatra/base'
+require './lib/chitter'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    erb :index
+  end
+
+  get '/peeps' do
+    @peeps = Peep.all
+    # @time = Peep.time
```

Comment:
It's always a good idea to remove commented out code from code you submit for code review (anywhere, not just at Makers)